lib/dns.js

```diff
@@ -82,11 +82,29 @@ function onlookup(err, addresses) {
 }
 
 
+function onlookupall(err, addresses) {
+  var i, results = [];
+  if (err) {
+    return this.callback(errnoException(err, 'getaddrinfo', this.hostname));
+  }
+
+  for (i = 0; i < addresses.length; i++) {
```

Comment:
Sure. Generally, I try to put `var` at the start of the scope, but i guess I'll follow the style of that file.